src/main/java/com/strangeone101/holoitemsapi/listener/ItemListener.java

```diff
@@ -1,790 +0,0 @@
-package com.strangeone101.holoitemsapi.listener;
```

Comment:
Deleting ItemListener seems like a bad idea if we still want to maintain the EventCache system. It also allows players to use Custom Items as their vanilla counterparts (e.g., using a custom item with a material string to craft a bow). It might be best to remove the parts we don't need.

Reply:
We may eventually need an EventCache system of some sort, but at the moment, we don't. And I am certain that we won't ever need it as it is currently implemented. If you go through all the source to see where it is used, you will find that EventHandlers build and update the cache, but there are no examples of items that demonstrate meaningful utilization of it. In that circumstance, it is likely it is half-baked. And if it ever comes to us having a use case, I would rather we make our own simpler implementation that better suits our enchantments based system than try to adapt Strange's, which was built with the ItemEvent system that will be essentially gone soon.

Reply:
What I mean by vanilla counterparts is that players can use the custom items as if they are not custom items.

Like the example I gave. If we have a custom item with the material string, that is meant to only be just a crafting material for other custom items, that string can be used for other vanilla purposes (making a bow/fishing rod). ItemListener and RecipeManager handles this.

Unless we want to reimplememt RecipeManager too, I think we should keep the crafting and inventory event handlers

Reply:
Got it now. I put back the parts that use the recipe manager.